### PR TITLE
feat: oct newsletter and on-call internal docs

### PR DESCRIPTION
**Pull request by nkrantz**
This PR adds the October newsletter to the Paste blog and adds the UX Infra on-call rotation docs to the repo's internal docs.

### CODE REVIEW

File: internal-docs/engineering/on-call-rotation.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+| ---------------- | --------------- | ----------------------------- | --------------------------------------- |
+| 0m               | 10m (+10m)      | 20m (+10m)                    | 50m (+30m)                              |
+
+## How do I best prepare for my rotation?
+
+- Make sure you’ve completed the PagerDuty training on Continu.
+- Check your [PagerDuty](https://twilio.pagerduty.com/) profile and ensure your contact information is up to date.
+- Get the [PagerDuty app](https://apps.apple.com/us/app/pagerduty/id594039512) set up on your phone, and follow processes to ensure that PagerDuty notifications will be classified as time-sensitive to bypass your phone’s do not disturb settings.
+- You can use the PagerDuty Slack integration to check who is on call at any point by typing ‘/fh help’ and hitting enter.
+- Review our [TODO: incident runbooks] to have a solid understanding of common production incidents and how to triage them.
```
Comment (nkrantz) [inline internal-docs/engineering/on-call-rotation.md:50]: Is this still a TODO or do we have it now? Is it the doc shadi made?

File: packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+        avatar
+        author_link
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
```
[the remark this answers is not in the document]
Comment (nkrantz) [reply packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx:40]: copy paste :) but localhost:8888__graphql lets you test out the queries if you were writing your own

File: internal-docs/engineering/on-call-rotation.md (hunk 1 above)
Comment (TheSisb) [reply internal-docs/engineering/on-call-rotation.md:50]: Runbooks are different. We only have 1 or 2 so far https://drive.google.com/drive/u/0/folders/1-_Xz9xtQ-NUA9kjNqw328g5OKH02swpa
There are tickets for more

File: packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+
+<Paragraph>
+  <Anchor href="/components/file-picker">Check out the File Picker docs</Anchor>
+</Paragraph>
+
+### Additional updates
+
+#### New
+
+- Inverse components in Figma (Anchor, Button, Combobox, Help Text, Input, and Select)!
```
Comment (shleewhite) [inline packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx:107]: Question: is there a link for this?

**Review by shleewhite — APPROVED** [review]
Looks good to me!

File: packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx (hunk 3 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx:107]: there wasn't one added to the newsletter 🤷

**Review by TheSisb — APPROVED** [review]

**Review by SiTaggart — APPROVED** [review]

**Review by kelseycavitt — APPROVED** [review]

**Review by shleewhite — APPROVED** [review]

**Pull request merged**